content/docs/setup/kubernetes/_index.md

```diff
@@ -2,13 +2,59 @@
 title: Kubernetes
 description: Instructions for installing the Istio control plane on Kubernetes and adding virtual machines into the mesh.
 weight: 10
-keywords: [kubernetes]
+type: section-index
 aliases:
     - /docs/tasks/installing-istio.html
     - /docs/setup/install-kubernetes.html
 icon: kubernetes
+keywords: [kubernetes, install, quick start, setup, installation]
```

Comment:
Here and elsewhere, there shoujldn't be any spaces in the keyword list.

Reply:
Why? What's the difference in the suggestions on the See also section? 
I ask because when I added the keywords with dashes, I got the impression that the section was not populating correctly. When I added them with spaces, the suggestions that appeared were better.

Reply:
I don't remember the reason just now, but putting spaces in there doesn't work right in some cases. So no spaces after the commas, and if you want a multi-word keyword then put a - instead of the space.

Search for keywords: in the doc repo to see all the other example usage to make sure you're using keywords consistent with the rest of the universe. This makes See Also better.

Reply:
Understood, adding the dashes.